File: tests/fixtures/python/policies/pii_redaction_before_llm/violating/app/handlers.py
import anthropic


def summarize_user(user_id: str) -> str:
    user = load_user(user_id)
    return anthropic.messages.create(model="claude", input=user)


def load_user(user_id: str) -> dict:
    return {"id": user_id, "email": "x@example.com"}

File: tests/fixtures/typescript/policies/pii_redaction_before_llm/violating/src/handlers.ts
import * as anthropic from 'anthropic';

export function summarizeUser(userId: string): string {
  const user = loadUser(userId);
  return anthropic.messages.create({ model: 'claude', input: user });
}

export function loadUser(userId: string): { id: string; email: string } {
  return { id: userId, email: 'x@example.com' };
}
